test/Tester/TestUtils.cs

```diff
@@ -114,20 +114,12 @@ public static class RequestContextTestUtils
     {
         public static void SetActivityId(Guid id)
         {
-#if NETSTANDARD
             RequestContext.ActivityId.Value = id;
```

Comment:
Created a PR against your branch (https://github.com/dVakulen/orleans/pull/8), let me know what you think. I believe it greatly simplifies the ActivityID API (and impl), especially for end users of both .NET and .NET Standard.